src/index.html

```diff
@@ -11,6 +11,43 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="search-bar"
+      data-qa="big"
+      action="#"
+      method="get"
+    >
+      <img
+        class="search-bar__loupe"
+        alt="search-loupe"
+        src="./images/Search.svg"
```

Comment:
The image tag is self-closing. It's good practice to close self-closing tags, e.g., <img ... />